src/template/page.ejs

```diff
@@ -18,7 +18,7 @@
     <%- headFileBottomContent %>
     <% if (faviconUrl) { %><link rel="icon" href="<%- faviconUrl %>"><% } %>
 </head>
-<body data-spy="scroll" data-target="#page-nav" data-offset="100">
+<body <%_ if (pageNav) { %> data-spy="scroll" data-target="#page-nav" data-offset="100" <%_ } %>>
```

Comment:
Within `Page.js`

```Javascript
    headFileTopContent: this.headFileTopContent,
    pageNav: this.frontMatter.pageNav, // Should change this to be this.isPageNavigationSpecifierValid
    siteNav: this.frontMatter.siteNav,
```

Because the current logic to insert a `page-nav` is through that function. Page navigation is not inserted if the specifier has wrong syntax.